spec/models/driver_spec.rb

```diff
@@ -64,8 +64,18 @@
   describe '.truck_name' do
     subject { driver.truck_name }
 
-    let(:driver) { create(:driver) }
+    context 'single truck' do
+      let(:driver) { create(:driver, trucks: trucks) }
+      let(:trucks) { [create(:truck)] }
 
-    it { is_expected.to eq(driver.trucks.map{ |truck| truck.license_plate }.join(', ')) }
+      it { is_expected.to eq("#{trucks.first.license_plate}") }
```

Comment:
no need for interpolation in this case
```suggestion
      it { is_expected.to eq(trucks.first.license_plate) }
```